users/views.py

```diff
@@ -60,23 +61,29 @@ def post(self, request):
 
                 token = RefreshToken.for_user(custom_user).access_token
                 current_site = get_current_site(request).domain
-                relative_link = reverse('send_email_confirmation')
-                abs_url = 'http://' + current_site + relative_link + '?token=' + str(token)
+                relative_link = reverse('verify-email', kwargs={'token': token, 'user_id': custom_user.id})
+                abs_url = 'http://'+ current_site + relative_link + '?token=' + str(token) + '?id=' + str(custom_user.id) 
                 email_body = 'Hi ' + custom_user.first_name + ' Use the link below to verify your email \n' + abs_url
-                sended_data = {'email_body': email_body, 'email_subject': 'Email confirmation',
-                               'to_email': custom_user.email}
-                # Util.send_email(data=sended_data)
-
-                return Response({"message": "User created successfully"}, status=status.HTTP_201_CREATED)
+                sended_data = {'email_body': email_body, 'email_subject': 'Email confirmation', 'to_email': custom_user.email}
+                Util.send_email(data=sended_data)
+                                
+                return Response({"User id": custom_user.id, "User name": custom_user.first_name, "message": "User created successfully"}, status=status.HTTP_201_CREATED)
             else:
                 return Response({"message": "Failed to create user"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
         else:
             return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 
-class VerifyEmailAPIView(APIView):
-    def get(self):
-        pass
+class SendEmailConfirmationAPIView(APIView):
+    def get(self, request, token=None, user_id=None):
```

Comment:
Consider using path parameters for `user_id` and `token` instead of query parameters if they are not already. This helps keep the URL cleaner and is more restful